install_pim/manual/system_requirements/manual_system_installation_debian10.rst

```diff
@@ -12,61 +12,49 @@ application to match your local installation.
 System installation
 -------------------
 
-MySQL 5.7
+MySQL 8.0
 *********
 
-The easiest way to install MySQL 5.7 is to use the official vendor package.
+The easiest way to install MySQL 8.0 is to use the official vendor package.
 
 First, start by installing the `MySQL APT repository <https://dev.mysql.com/doc/mysql-apt-repo-quick-guide/en/#apt-repo-setup>`_.
 It's a tool that aims to ease the installation and update of MySQL products.
-During the installation of this tool, one will be asked to choose the versions of the MySQL server to install. MySQL 5.7 has to be chosen.
+During the installation of this tool, one will be asked to choose the versions of the MySQL server to install. MySQL 8.0 has to be chosen.
 It's also recommended to disable all non desired tools, such as MySQL Workbench or MySQL Router.
 
 .. code-block:: bash
 
-    # apt install lsb-release apt-transport-https ca-certificates
-    # wget -O mysql-apt-config.deb https://dev.mysql.com/get/mysql-apt-config_0.8.7-1_all.deb
+    # apt install lsb-release apt-transport-https ca-certificates wget gnupg
+    # wget -O mysql-apt-config.deb https://dev.mysql.com/get/mysql-apt-config_0.8.13-1_all.deb
     # dpkg -i mysql-apt-config.deb
-    # apt install dirmngr
-    # apt-key adv --keyserver keys.gnupg.net --recv-keys 8C718D3B5072E1F5
 
 Now is the time to install what has been configured in the step before:
 
 .. code-block:: bash
 
-    # apt update
-    # apt install mysql-server
+    # apt update && apt install mysql-server
```

Comment:
apt => apt-get.
As apt has not yet a stable API, we should continue to use apt-get until apt is stable enough.